frontend/server/src/UITools.php

```diff
@@ -152,8 +153,10 @@ private static function getCommonPayload(
             'user' => $user,
             'is_admin' => $isAdmin,
             'associated_identities' => $associatedIdentities,
+            'api_tokens' => $apiTokens,
         ] = \OmegaUp\Controllers\Session::getCurrentSession();
-        return [
+        /** @var CommonPayload */
+        $commonPayload = [
```

Comment:
Parece que esta solución funciona para que se corrija el error, pero lo ideal sería regresar esto como estaba y mejor arreglar el problema de raiz:
```suggestion
        return [
```

Lo que tendríamos que hacer es buscar  la función de `Session::getCurrentSession` y añadirle el tipo correcto en la declaración de tipo en el `return`.

En concreto hay que agregar `api_tokens: list<ApiToken>` en la declaración de `CurrentSession`, y como en ese archivo no existe la declaración de `ApiToken`, habría que agregar la anotación un renglón arriba:
```php
 * @psalm-type ApiToken=array{name: string, timestamp: \OmegaUp\Timestamp, last_used: \OmegaUp\Timestamp, rate_limit: array{reset: \OmegaUp\Timestamp, limit: int, remaining: int}}
```

Reply:
Listo